an/v1/AN_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,2464 @@
+        vesselVoyageAtDestinationCountry:
+          $ref: '#/components/schemas/VesselVoyageArrivingAtDestinationCountry'
+        transports:
+          $ref: '#/components/schemas/Transports'
+        freetime:
+          description: |
+            List of freetime applicable to this shipment at destination.
+          type: array
+          items:
+            $ref: '#/components/schemas/Freetime'
+        charges:
+          description: |
+            List of charges applicable to this shipment.
+          type: array
+          items:
+            $ref: '#/components/schemas/Charge'
+        invoicePayableAt:
+          type: object
+          description: |
+            Location where payment of ocean freight and charges for the main transport will take place by the customer.
+
+            The location can be provided as a `UN Location Code` or as a fallback - a `freeText` field
+          oneOf:
+            - type: object
+              title: UN Location Code
+              properties:
+                UNLocationCode:
+                  type: string
+                  pattern: '^[A-Z]{2}[A-Z2-9]{3}$'
+                  minLength: 5
+                  maxLength: 5
+                  description: |
+                    The UN Location code specifying where the place is located. The pattern used must be
+
+                    - 2 characters for the country code using [ISO 3166-1 alpha-2](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en)
+                    - 3 characters to code a location within that country. Letters A-Z and numbers from 2-9 can be used
+
+                    More info can be found here: [UN/LOCODE](https://unece.org/trade/cefact/UNLOCODE-Download)
+                  example: NLAMS
+              required:
+                - UNLocationCode
+            - type: object
+              title: Free text
+              properties:
+                freeText:
+                  type: string
+                  maxLength: 35
+                  description: |
+                    The name of the location where payment will be rendered by the customer.
+                  example: DCSA Headquarters
+              required:
+                - freeText
+        utilizedTransportEquipments:
+          description: |
+            List of `Utilized Transport Equipment`.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/UtilizedTransportEquipment'
+        consignmentItems:
+          description: |
+            List of `Consignment`.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/ConsignmentItem'
+      required:
+        - issueDate
+        - carrierCode
+        - carrierCodeListProvider
+        - carrierContactInformation
+        - transportDocumentReference
+        - transportDocumentTypeCode
+        - isElectronic
+        - deliveryTypeAtDestination
+        - cargoMovmentTypeAtDesination
```

Comment:
Typo for cargoMovementTypeAtDestination. Because of which it is not shown as required.